watched/sounds.py

```diff
@@ -101,7 +101,7 @@ def skeletons(bot, sent_by, msg_text, channel=None):
 
 
 @messagehandler.register("twitch", "!quack")
-def skeletons(bot, sent_by, msg_text, channel=None):
```

Comment:
Skeletons are cool and all but I'm not all about dupe defs

Reply:
I need  a function name :)